addons/event/static/src/scss/event.scss

```diff
@@ -30,6 +30,12 @@
             min-height: 95px;
         }
     }
+    .attendee_badge{
+        font-size: 1.2rem;
+        padding: 10px 20px;
+        margin-top: -3px;
+        margin-right: -15px;
```

Comment:
there is something strange with those negative margins. It looks like they are added to reduce the padding we add here :) 

Then why not directly give the wanted paddings?

padding: 7px 5px 10px 20px; (to check)